src/solvers/parabolic_solvers.jl

```diff
@@ -0,0 +1,35 @@
+"""
+  struct BR1 end
```

Comment:
```suggestion
    BR1()
```
In general, I prefer avoiding too strict documentation of `struct`s. If we document it as you suggested, the exact struct layout becomes part of the public API and any changes will be breaking. If we do it as in my suggestion, we only document a constructor but the struct itself can be changed later without problems.